### PR TITLE
Add query Webhook events

### PR DESCRIPTION
**Pull request by korycins**
This pr adds the query to fetch all webhook events with their names.

### CODE REVIEW

**Comment by django-queries** [comment]
Here is the report for ef4f4b92f408d5d028f77ab5e04d3f643df25054 (korycins/saleor @ fix/add_webhook_event_query_to_fetch_all_events)
Base comparison is 1218f4ff14811c2145775df76cc06e18b37132b7.



<details><summary>No differences were found. (click me)</summary>
<p>

```diff
# api.benchmark checkout
  test name                                  	left count 	right count	duplicate count
  -------------------------------------------	-----------	-----------	---------------
  add billing address to checkout            	         34	         34	             20
  add shipping to checkout                   	          7	          7	              0
  checkout payment charge                    	         14	         14	              0
  complete checkout                          	          6	          6	              0
  create checkout                            	         48	         48	             24

# api.benchmark homepage
  test name                                  	left count 	right count	duplicate count
  -------------------------------------------	-----------	-----------	---------------
  retrieve main menu                         	          5	          5	              0
  retrieve product list                      	          4	          4	              0
  retrieve secondary menu                    	          5	          5	              0
  retrieve shop                              	          2	          2	              0

# api.benchmark product
  test name                                  	left count 	right count	duplicate count
  -------------------------------------------	-----------	-----------	---------------
  product details                            	         15	         15	              3
  retrieve product attributes                	         13	         13	              2

# api.benchmark variant
  test name                                  	left count 	right count	duplicate count
  -------------------------------------------	-----------	-----------	---------------
  retrieve variant list                      	         15	         15	              6

# api product sorting attributes
  test name                                  	left count 	right count	duplicate count
  -------------------------------------------	-----------	-----------	---------------
  sort product not having attribute data     	         21	         21	              0
```

</p>
</details>

File: saleor/graphql/webhook/types.py [hunk 1]
```diff
@@ -1,16 +1,24 @@
...
-from ...webhook import models
+from ...webhook import WebhookEventType, models
 from ..core.connection import CountableDjangoObjectType
 from .enums import WebhookEventTypeEnum
 
 
-class WebhookEvent(graphene.ObjectType):
-    event_type = WebhookEventTypeEnum(description="Name of the event type.")
+class WebhookEvent(CountableDjangoObjectType):
+    name = graphene.String(description="Printable name of the event")
```
Comment (maarcingebala) [inline saleor/graphql/webhook/types.py]:
```suggestion
    name = graphene.String(description="Display name of the event.")
```

File: saleor/graphql/webhook/types.py [hunk 2]
```diff
@@ -1,16 +1,24 @@
...
+    event_type = WebhookEventTypeEnum(description="Internal name of the event type.")
 
     class Meta:
+        model = models.WebhookEvent
         description = "Webhook event."
+        only_fields = ["event_type", "name"]
+
+    @staticmethod
+    def resolve_name(info: models.WebhookEvent, *_args, **_kwargs):
+        name = WebhookEventType.DISPLAY_LABELS.get(info.event_type) or info.event_type
```
Comment (maarcingebala) [inline saleor/graphql/webhook/types.py]: No need to assign to a variable if you don't use it.

File: saleor/graphql/webhook/types.py [hunk 3]
```diff
@@ -1,16 +1,24 @@
...
+    name = graphene.String(description="Printable name of the event")
+    event_type = WebhookEventTypeEnum(description="Internal name of the event type.")
 
     class Meta:
+        model = models.WebhookEvent
         description = "Webhook event."
+        only_fields = ["event_type", "name"]
+
+    @staticmethod
+    def resolve_name(info: models.WebhookEvent, *_args, **_kwargs):
```
Comment (maarcingebala) [inline saleor/graphql/webhook/types.py]: The first argument should be `root`, `info` is the second argument.

**Pull request merged**